### PR TITLE
Flake8 docstrings complete

### PR DESCRIPTION
**Pull request by jdkandersson**
Introduces the `flake8-docstrings-complete` and `flake8-test-docs` linters and fixes the issues raised by the linters which are all in the docstrings

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -128,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VSCode
+.vscode
```
Comment (yanksyoon) [inline .gitignore:133]: I'm wondering if we should use a global .gitignore to ignore IDE-related artifacts. What do you think?

**Review by yanksyoon — APPROVED** [review]
LGTM, just left a minor question regarding IDE artifacts! :)

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Pull request merged**